### ISSUE TITLE
Data race warning in peer_event_loop.go

### ISSUE THREAD

**Issue by lignay** [matrixorigin/matrixcube#179]
#### Can be reproduced ?
yes

#### Steps:
Refer to https://github.com/matrixorigin/matrixone/runs/3411122918?check_suite_focus=true  for details.

Warning info snippet as below:

WARNING: DATA RACE
Read at 0x00c0048c8068 by goroutine 94:
  github.com/matrixorigin/matrixcube/raftstore.(*peerReplica).onRaftTick()
      /home/liyang/go/pkg/mod/github.com/matrixorigin/matrixcube@v0.0.0-20210824024740-556715aafe65/raftstore/peer_event_loop.go:116 +0x4d
  github.com/matrixorigin/matrixcube/raftstore.(*peerReplica).onRaftTick-fm()
      /home/liyang/go/pkg/mod/github.com/matrixorigin/matrixcube@v0.0.0-20210824024740-556715aafe65/raftstore/peer_event_loop.go:115 +0x4d
  github.com/fagongzi/goetty/timewheel.(*TimeoutWheel).doExpired()
      /home/liyang/go/pkg/mod/github.com/fagongzi/goetty@v1.9.0/timewheel/timewheel.go:382 +0x497
  github.com/fagongzi/goetty/timewheel.(*TimeoutWheel).Start·dwrap·3()
      /home/liyang/go/pkg/mod/github.com/fagongzi/goetty@v1.9.0/timewheel/timewheel.go:220 +0x39

Previous write at 0x00c0048c8068 by goroutine 473:
  github.com/matrixorigin/matrixcube/raftstore.(*peerReplica).handleReady()
      /home/liyang/go/pkg/mod/github.com/matrixorigin/matrixcube@v0.0.0-20210824024740-556715aafe65/raftstore/peer_event_raft_ready.go:97 +0x964
  github.com/matrixorigin/matrixcube/raftstore.(*peerReplica).handleEvent()
      /home/liyang/go/pkg/mod/github.com/matrixorigin/matrixcube@v0.0.0-20210824024740-556715aafe65/raftstore/peer_event_loop.go:190 +0x284
  github.com/matrixorigin/matrixcube/raftstore.(*store).runPRTask.func1.1()
      /home/liyang/go/pkg/mod/github.com/matrixorigin/matrixcube@v0.0.0-20210824024740-556715aafe65/raftstore/store.go:432 +0xdc
  sync.(*Map).Range()
      /home/liyang/stage/actions-runner/_work/_tool/go/1.17.0/x64/src/sync/map.go:346 +0x205
  github.com/matrixorigin/matrixcube/raftstore.(*store).runPRTask.func1()
      /home/liyang/go/pkg/mod/github.com/matrixorigin/matrixcube@v0.0.0-20210824024740-556715aafe65/raftstore/store.go:430 +0x8f
  github.com/matrixorigin/matrixcube/raftstore.(*store).runPRTask()
      /home/liyang/go/pkg/mod/github.com/matrixorigin/matrixcube@v0.0.0-20210824024740-556715aafe65/raftstore/store.go:451 +0x1d3
  github.com/matrixorigin/matrixcube/raftstore.(*store).startRaftWorkers.func1()
      /home/liyang/go/pkg/mod/github.com/matrixorigin/matrixcube@v0.0.0-20210824024740-556715aafe65/raftstore/store.go:417 +0x84
  github.com/fagongzi/util/task.(*Runner).RunCancelableTask.func1()
      /home/liyang/go/pkg/mod/github.com/fagongzi/util@v0.0.0-20210409031311-a10fdf8fbd7a/task/task.go:350 +0xcf
.....

#### Actual behavior:
n/a

#### Environment:
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:

#### Additional context:
- Error message from client:
- Server log:
- Other information:

**Comment by lignay** [comment]
Please have a look at this race warning, need to figure out what's the root cause.

**Comment by lni** [comment]
Code is below. 

```
if !pr.stopRaftTick {
  // When we apply snapshot, stop raft tick and resume until the snapshot applied
  pr.stopRaftTick = true
}
```

```
if !pr.stopRaftTick {
  ...
}
```

This is of minimal impact. Merged into  #128.
